aspnetcore/security/enforcing-ssl.md

```diff
@@ -345,36 +301,6 @@ For chromium browsers on Linux:
 
 <a name="trust-ff-linux"></a>
 
-#### Trust the certificate with Firefox on Linux
```

Comment:
Why delete, rather than update, this section?  It seems like we could just tell people to run `certutil` as for Chromium.

Reply:
There's a note in an unedited part of this document (above) that says `certutil` is "legacy".  I'm not sure why - maybe just the specific version linked?

Reply:
Above, where it says to run `certutil` twice, it should probably only be run once - with `"P,,"` for Chromium or `"C,,"` for Firefox.

Reply:
> Why delete, rather than update, this section? It seems like we could just tell people to run `certutil` as for Chromium.

Are the instructions different using  `certutil`  on FF different than for Chromium? If not, we'll make sure the Chromium instructions say Chromium and FF.

Reply:
> There's a note in an unedited part of this document (above) that says `certutil` is "legacy". I'm not sure why - maybe just the specific version linked?

the *legacy* comment is only for the the [Red Hat Enterprise Linux tab](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-8.0&branch=pr-en-us-33472&tabs=visual-studio%2Clinux-rhel#trust-https-certificate-on-linux-using-edge-or-chrome)

Reply:
> Are the instructions different using certutil on FF different than for Chromium? If not, we'll make sure the Chromium instructions say Chromium and FF.

The two differences are the usage, as noted in my comment, and the location of the database.  Otherwise, it should be the same.